apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,25 +154,14 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
-      strategy.hasDeterminativeProgress !== undefined &&
-      strategy.hasDeterminativeProgress();
-  };
-
-  //
   // Shows the progress dialog based on the
   //
   var _displayProgress = function _displayProgress() {
-    var progressClass = determinativeProgress ? 'progressBar' : 'spinner';
 
     Contacts.utility('Overlay', function _loaded() {
       progress = utils.overlay.show(
         strategy.getExportTitle(),
-        progressClass,
+        true,
```

Comment:
Looks a bit optimistic to pass Boolean value into the function instead of String. Do we have a direct requirement to use onlly one type of progress indicator for exporting contacts? I'm asking because as far as I can see, the original logic for choosing between 'spinner' and 'progress bar' being removed by this commit.